build.gradle.kts

```diff
@@ -9,6 +11,37 @@ configurations {
 
 dependencies {
   implementation("org.springframework.boot:spring-boot-starter-webflux")
+  annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
+
+  implementation("org.springframework.boot:spring-boot-starter-webflux")
+  implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+
+  implementation("org.springframework.boot:spring-boot-starter-security")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
+
+  implementation("org.flywaydb:flyway-core")
+  runtimeOnly("org.postgresql:postgresql:42.2.23")
+  runtimeOnly("com.h2database:h2:1.4.200")
```

Comment:
do you need h2?

Reply:
Probably not now switched to testcontainers and postgres - will remove